seatunnel-dist/release-docs/LICENSE

```diff
@@ -263,64 +276,120 @@ The text of each license is the standard Apache 2.0 license.
      (Apache 2.0 License) Spark Project Streaming (org.apache.spark:spark-streaming_2.11:2.4.0 - http://spark.apache.org/)
      (Apache 2.0 License) Spark Project Tags (org.apache.spark:spark-tags_2.11:2.4.0 - http://spark.apache.org/)
      (Apache 2.0 License) Spark Project Unsafe (org.apache.spark:spark-unsafe_2.11:2.3.4 - http://spark.apache.org/)
+     (Apache 2.0 License) Spark Project Unsafe (org.apache.spark:spark-unsafe_2.11:2.4.0 - http://spark.apache.org/)
      (Apache 2.0 License) TiSpark Project Assembly (com.pingcap.tispark:tispark-assembly:2.4.1 - http://github.copm/pingcap/tispark)
      (Apache 2.0 License) TiSpark Project Core Internal (com.pingcap.tispark:tispark-core-internal:2.4.1 - http://github.copm/pingcap/tispark)
      (Apache 2.0 License) TiSpark Project TiKV Java Client (com.pingcap.tikv:tikv-client:2.4.1 - http://github.copm/pingcap/tispark/tikv-client)
      (Apache 2.0) Moshi (com.squareup.moshi:moshi:1.8.0 - https://github.com/square/moshi/moshi)
+     (Apache 2.0) OkHttp (com.squareup.okhttp:okhttp:1.0.2 - https://github.com/square/okhttp/okhttp)
+     (Apache 2.0) OkHttp (com.squareup.okhttp:okhttp:2.4.0 - https://github.com/square/okhttp/okhttp)
+     (Apache 2.0) Okio (com.squareup.okio:okio:1.4.0 - https://github.com/square/okio/okio)
      (Apache 2.0) error-prone annotations (com.google.errorprone:error_prone_annotations:2.3.4 - http://nexus.sonatype.org/oss-repository-hosting.html/error_prone_parent/error_prone_annotations)
+     (Apache 2.0) error-prone annotations (com.google.errorprone:error_prone_annotations:2.8.0 - https://errorprone.info/error_prone_annotations)
```

Comment:
Please check these duplicated dependencies with different versions, normally these should not be the case.

Reply:
Okay, you mean the license-maven-plugin normally should merge them, right